sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/_input_output_helpers.py

```diff
@@ -309,6 +309,8 @@ def to_rest_data_outputs(outputs: Dict[str, Output]) -> Dict[str, RestJobOutput]
             output_value_type = output_value.type if output_value.type else AssetTypes.URI_FOLDER
             if output_value_type in target_init_func_dict:
                 output = target_init_func_dict[output_value_type](
+                    output_value.asset_name,
```

Comment:
Should we fetch name and version inside from_rest_data_outputs() as well?

Reply:
added